src/pudl/transform/vcerare.py

```diff
@@ -363,21 +385,30 @@ def check_rows(context: AssetCheckExecutionContext) -> AssetCheckResult:
     """Check rows."""
     logger.info("Check VCE RARE hourly table is the expected length")
 
-    # Define row counts for fast/full etl
-    # TODO 2024-12-27: make this check row counts per year instead of having
-    # two different counts based on job name - less brittle.
-    row_counts = {
-        "etl_full": 136437000,
-        "etl_fast": 27287400,
+    # Define row counts for report years
+    row_counts_by_year = {
+        2019: 27287400,
+        2020: 27287400,
```

Comment:
Guess they don't take leap years into account 🤷

Reply:
Yes that's right --VCE is an 8760 dataset. No December 31st data in leap-years.